modules/farm_fd2/src/entrypoints/transplanting/lib.submit.unit.cy.js

```diff
@@ -0,0 +1,221 @@
+      'length'
+    );
+    expect(result.transplantingBedFeetQuantity.attributes.label).to.equal(
+      'Bed Feet'
+    );
+  });
+
+  it('Check the rows/bed quantity--standard', () => {
+    expect(
+      result.transplantingRowsPerBedQuantity.attributes.value.decimal
+    ).to.equal(form.rowsPerBed);
+    expect(result.transplantingRowsPerBedQuantity.type).to.equal(
+      'quantity--standard'
+    );
+    expect(result.transplantingRowsPerBedQuantity.attributes.measure).to.equal(
+      'ratio'
+    );
+    expect(result.transplantingRowsPerBedQuantity.attributes.label).to.equal(
+      'Rows/Bed'
+    );
+  });
+
+  it('Check the row feet quantity--standard', () => {
+    expect(
+      result.transplantingRowFeetQuantity.attributes.value.decimal
+    ).to.equal(form.rowsPerBed * form.bedFeet);
+    expect(result.transplantingRowFeetQuantity.type).to.equal(
+      'quantity--standard'
+    );
+    expect(result.transplantingRowFeetQuantity.attributes.measure).to.equal(
+      'length'
+    );
+    expect(result.transplantingRowFeetQuantity.attributes.label).to.equal(
+      'Row Feet'
+    );
+  });
+
+  it('Check the bed width quantity--standard', () => {
+    expect(
+      result.transplantingBedWidthQuantity.attributes.value.decimal
+    ).to.equal(form.bedWidth);
+    expect(result.transplantingBedWidthQuantity.type).to.equal(
+      'quantity--standard'
+    );
+    expect(result.transplantingBedWidthQuantity.attributes.measure).to.equal(
+      'length'
+    );
+    expect(result.transplantingBedWidthQuantity.attributes.label).to.equal(
+      'Bed Width'
+    );
+  });
+
+  it('Check the log--activity', () => {
+    expect(result.transplantingLog.attributes.name).to.equal(
+      form.transplantingDate + '_xp_' + form.picked[0].data.crop
+    );
+    expect(
+      fieldMap.get(result.transplantingLog.relationships.location[0].id)
+        .attributes.name
+    ).to.equal(form.location);
+    expect(
+      bedMap.get(result.transplantingLog.relationships.location[1].id)
+        .attributes.name
+    ).to.equal(form.beds[0]);
+    expect(
+      bedMap.get(result.transplantingLog.relationships.location[2].id)
+        .attributes.name
+    ).to.equal(form.beds[1]);
+  });
+
+  it('Check the depth quantity--standard', () => {
+    expect(result.depthQuantity.attributes.value.decimal).to.equal(form.depth);
+    expect(result.depthQuantity.type).to.equal('quantity--standard');
+    expect(result.depthQuantity.attributes.measure).to.equal('length');
+    expect(result.depthQuantity.attributes.label).to.equal('Depth');
+  });
+
+  it('Check the speed quantity--standard', () => {
+    expect(result.speedQuantity.attributes.value.decimal).to.equal(form.speed);
+    expect(result.speedQuantity.type).to.equal('quantity--standard');
+    expect(result.speedQuantity.attributes.measure).to.equal('rate');
+    expect(result.speedQuantity.attributes.label).to.equal('Speed');
+  });
+
+  it('Check the soil disturbance log--activity', () => {
```

Comment:
Seems like there should be more tests here.  Have a look at the `farmosUtil.js` function for creating a soil disturbance log. Then make sure that each of the parameters passed has an assertion in the test.  That will ensure that the correct values were passed from `lib.js` to the `farmosUtil.js` function.